src/routes/login/containers/LoginPage.js

```diff
@@ -1,21 +1,16 @@
-const Connect = require('react-redux').connect;
+const { useMiddleEnd } = require('strange-middle-end');
 const LoginPage = require('../components/LoginPage');
-const M = require('../../../middle-end');
 
-const internals = {};
+module.exports = function LoginPageContainer(props) {
 
-internals.connect = Connect(
-    (state) => ({
-        isAuthenticated: M.selectors.auth.getIsAuthenticated(state)
-    }),
-    (dispatch) => ({
-        reqCreateAccount: async ( loginInfo ) => {
+    const M = useMiddleEnd();
 
-            const [err, result] = await M.dispatch.auth.login(loginInfo);
+    const onPressLogin = async ( accountInfo ) => {
```

Comment:
Can't suggest this since it's interleaved with deleted lines, but something like this would work equally well:
```js
const onPressLogin = (accountInfo) => m.dispatch.auth.login(accountInfo);
```